### PR TITLE
What: Due Diligence of members and Disclaimers on content by the grou…

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
…p #277 closeout

Why:

* Update contributing doc to include security issues from security reviews

This change addresses the need by:

* adding header & content for contributions of security issues.

### CODE REVIEW

**Comment by TheFoxAtWork** [comment]
@lumjjb @achetal01 @anvega @ashutosh-narkar could someone review so we can get this merged in?

File: CONTRIBUTING/README.md [hunk 1]
```diff
@@ -34,4 +34,15 @@ and commenting on issues is a great way to get involved!
 
 When creating or reviewing pull requests, please refer to the
 [writing style guide](writing-style.md) to help maintain consistency across
-all of our documents.
\ No newline at end of file
+all of our documents.
+
+## Security Issues
```
[the remark this answers is not in the document]
Comment (ashutosh-narkar) [reply CONTRIBUTING/README.md:39]: Nit:  Can we think of a different name for this section like `Potential Security Vulnerabilities` or `Reporting Security Vulnerabilities` or `Reporting Security Issues`

**Review by ashutosh-narkar — APPROVED** [review]
LGTM

File: CONTRIBUTING/README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (TheFoxAtWork) [reply CONTRIBUTING/README.md]: fixed

[the remark this answers is not in the document]
Comment (ashutosh-narkar) [reply CONTRIBUTING/README.md]: Thanks!

**Comment by TheFoxAtWork** [comment]
Merging to get this pushed in, if there are additional discussions we can open a separate PR.

**Pull request merged**